src/ui/layout/mct-tree.vue

```diff
@@ -231,6 +232,9 @@ export default {
             if (!this.isLoading) {
                 this.setContainerHeight();
             }
+        },
+        ancestors() {
+            this.observeAncestors();
```

Comment:
I am alittle unsure of why this is a computed property? Computed properties should return something, if we need to do something when a value changes, we should use watch

Reply:
Oh wow, that was DEFINITELY supposed to be a watched property... I'll update.

Reply:
Ah, looks like it was watched, just looked weird in the code preview